### PR TITLE
Fixing #192

### PR DESCRIPTION
**Pull request by cchristelis**
Submit github issue

### CODE REVIEW

**Comment by cchristelis** [comment]
@dimasciput, will you please verify this.

**Comment by dimasciput** [comment]
@cchristelis yes the javascript works.

**Comment by timlinux** [comment]
Did someone confirm we can actually submit an issue with this fix in place?

File: django_project/github_issue/static/js/github-issue.js [hunk 1]
```diff
@@ -1,5 +1,5 @@
 //noinspection JSUnusedGlobalSymbols
-function submitIssue() {
+window.submitIssue = function () {
   $("#issue-submit-button").attr("disabled", "disabled");
   $.post("/github-issue", {
```
Comment (dimasciput) [inline django_project/github_issue/static/js/github-issue.js]: This url won't work, you need to remove the slash, so it will be `$.post("github-issue")`.

**Comment by dimasciput** [comment]
@timlinux just need minor fix ( see my comment above ), after it fixed everything works fine. And you will need to put GITHUB_USER, GITHUB_PASSWORD, GITHUB_URL in settings file.

**Comment by timlinux** [comment]
@dimasciput ok cool. I tried to disable LGTM again but it seems to be stuck :-P

**Comment by timlinux** [comment]
LGTM

**Comment by timlinux** [comment]
BTW @dimasciput it would be good to update the readme with instructions for issue submitting (if it is not already there). I guess it should go in secret.py....nice to have those in a template secret.py.templ (again I didn't check if it is already there).

**Pull request merged**